test/functional/rpc_fundrawtransaction.py

```diff
@@ -793,7 +793,7 @@ def test_option_feerate(self):
 
         self.log.info("Test fundrawtxn with invalid estimate_mode settings")
         for k, v in {"number": 42, "object": {"foo": "bar"}}.items():
-            assert_raises_rpc_error(-3, "Expected type string for estimate_mode, got {}".format(k),
+            assert_raises_rpc_error(-3, "JSON value of type {} is not of expected type string".format(k),
```

Comment:
Could update to f-strings while touching?

Reply:
Wouldn't be better if this is refactored in another issue/branch? There are several test files which don't use f-strings.

Reply:
No, I was only suggesting to use f-strings for the lines that you are touching anyway. The others can be kept as-is